libdrgn/python/program.c

```diff
@@ -477,6 +477,75 @@ static struct drgn_error *py_object_find_fn(const char *name, size_t name_len,
 	return drgn_object_copy(ret, &((DrgnObject *)obj)->obj);
 }
 
+static struct drgn_error *py_symbol_find_fn(const char *name, uint64_t addr,
+					    struct drgn_module *module,
+					    enum drgn_find_symbol_flags flags,
+					    void *data, struct drgn_symbol_result_builder *builder)
+{
+	struct drgn_error *err = NULL;
```

Comment:
```suggestion
	struct drgn_error *err;
```

Reply:
In this case it would be possible to return an uninitialized `err` if the loop body never executed (because the Python code returned an empty list).

But I think the correct solution is to explicitly `return NULL` at the end of this function, and move the declaration of `err` into the body of the loop since it is only used there.